tests/drivers/uart/uart_async_api/src/test_uart_async.c

```diff
@@ -324,17 +324,24 @@ ZTEST_USER(uart_async_multi_rx, test_multiple_rx_enable)
 }
 
 #if NOCACHE_MEM
-static __aligned(32) uint8_t chained_read_buf[2][8] __used __NOCACHE;
+/* To ensure 32-bit alignment of the buffer array,
+ * the two arrays are defined instead using an array of arrays
+ */
+static __aligned(32) uint8_t chained_read_buf_0[8] __used __NOCACHE;
```

Comment:
I have a question that's unrelated to your changes, does `__used` have any utility here?

Reply:
@JarmouniA ,  __used is to force compiler to emit symbol, but I do not know whether it is a must in Zephyr, so I copy the line headless.